tsconfig.json

```diff
@@ -1,12 +1,30 @@
 {
-  "extends": "@gplane/tsconfig",
```

Comment:
Why remove the config? Extending an existing config can reduce the complexity of `tsconfig.json`, instead of adding everything like below.

Reply:
The config matches Next.js exactly and Next.js manages the options automatically.

Reply:
Do you mean those options shouldn't be edited manually?

Reply:
> Do you mean those options shouldn't be edited manually?

ping @timneutkens

Reply:
For the most part yeah, some can be edited but those that don't match compilation will be changed back on boot otherwise.